dbms/src/Storages/Page/V3/BlobStore.cpp

```diff
@@ -1101,7 +1085,7 @@ PageEntriesEdit BlobStore::gc(std::map<BlobFileId, PageIdAndVersionedEntries> &
                 file_offset,
                 data_size,
                 total_page_size);
-            blob_file->write(data_beg, file_offset, data_size, write_limiter, /*background*/ true);
```

Comment:
`data_beg` is a name that takes me some time to think what it means because `beg` is a valid word and it's a little difficult to imagine that it's the abbreviation of `begin` . `data_pos` or `data_begin` is both ok for me.

Reply:
`data_begin` LGTM